src/components/footer/AboutModal.tsx

```diff
@@ -28,6 +29,9 @@ const AboutModal = (props: ModalProps): React.ReactElement => {
       <Modal.Body>
         <p dangerouslySetInnerHTML={{ __html: t('What_Is_Apertium') }} />
         <p dangerouslySetInnerHTML={{ __html: t('Maintainer') }} />
+        {Config.shouldDisplayString && (
+          <p dangerouslySetInnerHTML={{ __html: t('More_Languages') }} style={{ paddingBottom: '2em' }} />
```

Comment:
Rather than specific padding values, can you use Bootstrap's utility spacing classes?